config/logger_test.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/hazelcast/hazelcast-go-client/logger"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCustomLogger(t *testing.T) {
```

Comment:
as an addition to this test, can you write an integration test, something like this:

1. set --log-file and --log-level flags programatically
2. log several messages
3. assert that there is a file created and it includes expected content


is this possible?

Reply:
This required a bit of test infrastructure. Because of that, I implemented a test case somewhat similar to the one you described in this PR. If that's okay with you too we can follow this on that PR https://github.com/hazelcast/hazelcast-commandline-client/pull/141/files#diff-1baa2c4ec2628b94e24a5ce7123286a892a8f38f586444abd8ed7867b44cac3bR21

Reply:
Sure. I did not see a `--log-file` test but it looks ok.